packages/styles/scss/components/card-group/_card-group.scss

```diff
@@ -162,6 +161,68 @@
     }
   }
 
+  :host(#{$dds-prefix}-card-group)[border] {
+    &::after {
+      content: none;
+      box-shadow: none;
+    }
+
+    ::slotted(#{$dds-prefix}-card-group-item) {
+      border-right: 1px solid $ui-03;
```

Comment:
Is this work overwriting what Ignacio, is doing over in #6133? Or is it going to leverage that same work?

Reply:
Everything is contained inside the border attribute, it shouldn't affect his work.

Reply:
But, why not leverage his work. Why maintain the same border in two places?

Reply:
From what I discussed with @IgnacioBecerra, his cards only get a border if they don't have a href. So, for example, if you fill a card group with cards with links in them, the border would't be visible without this new attribute.

Reply:
> his cards only get a border if they don't have a href

I don't think that's supposed to be the case. The border is supposed to be optional for the card as a whole.

Reply:
> The border is supposed to be optional for the card as a whole.

If this is the case then shouldn't @IgnacioBecerra be leveraging this work, since his PR is specifically targeting card without an `href`, whereas this is for `card-group-item` in general.

Reply:
We're working together on making sure things don't overlap, just waiting for some design confirmation.